EmbeddedSocialClient/sdk/build.gradle

```diff
@@ -22,11 +22,6 @@ repositories {
     }
 }
 android {
-    signingConfigs {
-        debug {
-            storeFile file('debug.keystore')
-        }
-    }
     publishNonDefault true
```

Comment:
Do you still need line 39   `signingConfig signingConfigs.debug` ?

Reply:
This should not be necessary in the sdk's build.gradle. It is necessary in the app build flavor's build.gradle.